### ISSUE TITLE
Feature: expose the ecosystem name in Packaging check

### ISSUE THREAD

**Issue by laurentsimon** [ossf/scorecard#2494]
We currently don't expose the ecosystem name - raw results and in the logs.

**Comment by spencerschrock** [comment]
copying from #2495

technically the name could be more complex than just the repo URI, we would need to read `go.mod`. 
e.g. scorecard would report `github.com/ossf/scorecard` even though it's now `github.com/ossf/scorecard/v4`.

running it on `https://github.com/golang/tools/` (which is a mirror) would report that instead of the module name `golang.org/x/tools`

**Comment by laurentsimon** [comment]
Agreed that parsing go.mod is the right approach. Starting with the ecosystem name like `Go`, `PyPi` would be useful. Realistically, we won't be able to infer package names for other ecosystems unless the registry exposes it after verifying it themselves. Right?

**Comment by azeemshaikh38** [comment]
What is the motivation for exposing the ecosystem name in Packaging check? Also, how do we handle monorepos?

**Comment by laurentsimon** [comment]
the check currently looks for ecosystem-specific Actions, so we *already* surface this implcitely. If we find multiple Actions, we can identify multiples projects in monorepos.
